### PR TITLE
Added chat with admin

### LINKED ISSUE #7051: [GreenCityChat] Chat with Admin
As a User, I want to have an easy access to chat for locations, so that I can get informational support about order or delivery.

Preconditions:
User is registered
User is on UBS Courier page.
Business rules:
Each chat can have one or more assigned consultants.
Constraints:
Assignment and management questions from chat between multiple employees.

### PR DESCRIPTION
**Pull request by staaankey**
# GreenCityUBS PR

## Summary Of Changes :fire:
This pull request introduces a significant enhancement to the Employee entity by adding a boolean has_chat field. This field facilitates the management of access to chat support for one or multiple chats with tariffs. The update also includes necessary database schema changes reflected in ch-add-column-tariff-infos-mapping-Klopov.xml.

Additionally, new API endpoints have been implemented to support this functionality:

This enhancement enhances the flexibility and control over chat support access within the system, improving user experience.

## Issue Link :clipboard:
[Link on your task from board](https://github.com/ita-social-projects/GreenCity/issues/7051)

## Added
**New endpoints:**
- `"/check-if-tariff-exists/{id}"` that accept id of tariff and return Boolean type.
- `"/locations"` that returns list of active locations
- `"/tariffs/{locationId}"` that accept location id and return active tariff that corresponding to that location

**New migrations:**
- `ch-add-column-tarriff-infos-mapping-Klopov.xml`

**New tests that will cover new functionality to check if that works as expected**

## Changed
Added boolean `has_chat` field to Entity Employee with many to many relationship to enable/disable access to the chat support for one or multiple chats with tariff.

## Deleted
* _&lt;detail item of what was deleted&gt;_

## How to test :clipboard:

# PR Checklist Forms

_(to be filled out by PR submitter)_
- [x] Code is up-to-date with the `dev` branch.
- [x] You've successfully built and run the tests locally.
- [x] There are new or updated unit tests validating the change.
- [x] JIRA/ Github Issue number & title in PR title (ISSUE-XXXX: Ticket title)
- [x] This template filled (above this section).
- [x] Sonar's report does not contain bugs, vulnerabilities, security issues, code smells ar duplication
- [x] `NEED_REVIEW` and `READY_FOR_REVIEW` labels are added.
- [x] All files reviewed before sending to reviewers

### CODE REVIEW

**Review by ryhaldenys — APPROVED** [review]

**Review by ABbondar — APPROVED** [review]

File: core/src/main/java/greencity/configuration/SecurityConfig.java [hunk 1]
```diff
@@ -178,7 +180,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 UBS_LINK + "/order/{id}/cancellation",
                 ADMIN_LINK + "/notification/get-all",
                 ADMIN_LINK + "/notification/{id}",
-                ADMIN_LINK + "/**",
+                // TODO: ADMIN_LINK + "/**",
```
Comment (DenysLiubchenko) [inline core/src/main/java/greencity/configuration/SecurityConfig.java]: I think we do not need to commit todo

File: core/src/main/java/greencity/controller/ManagementEmployeeController.java [hunk 2]
```diff
@@ -48,26 +46,26 @@ public class ManagementEmployeeController {
...
-     * Controller saves employee.
+     * Controller method to save an employee.
      *
-     * @param employeeWithTariffsIdDto {@link EmployeeWithTariffsIdDto}
-     * @return {@link EmployeeWithTariffsDto} saved employee.
-     * @author Mykola Danylko.
+     * @param employeeWithTariffsIdDto DTO for {@link EmployeeWithTariffsIdDto}.
+     * @param image                    Image of the employee (optional).
+     * @return ResponseEntity with {@link EmployeeWithTariffsDto} instance.
+     * @author [Author Name]
```
Comment (DenysLiubchenko) [inline core/src/main/java/greencity/controller/ManagementEmployeeController.java]: Set [Author Name]

File: service-api/src/main/java/greencity/dto/employee/EmployeeDto.java [hunk 3]
```diff
@@ -3,25 +3,20 @@
...
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-import javax.validation.constraints.Email;
-import javax.validation.constraints.Min;
-import javax.validation.constraints.NotBlank;
-import javax.validation.constraints.NotEmpty;
-import javax.validation.constraints.NotNull;
-import javax.validation.constraints.Pattern;
+import javax.validation.constraints.*;
```
Comment (DenysLiubchenko) [inline service-api/src/main/java/greencity/dto/employee/EmployeeDto.java]: We do not use wildcart imports

File: service/src/main/java/greencity/mapping/employee/EmployeeUpdateDtoMapper.java [hunk 4]
```diff
@@ -30,10 +29,10 @@ protected EmployeeWithTariffsIdDto convert(Employee employee) {
...
                     .collect(Collectors.toList()))
                 .build())
-            .tariffId(employee.getTariffInfos()
-                .stream()
-                .map(TariffsInfo::getId)
-                .collect(Collectors.toList()))
+            // .tariffId(employee.getTariffs()
+            // .stream()
+            // .map(TariffsInfo::getId)
+            // .collect(Collectors.toList()))
```
Comment (DenysLiubchenko) [inline service/src/main/java/greencity/mapping/employee/EmployeeUpdateDtoMapper.java]: Commiting comented code

File: service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java [hunk 5]
```diff
@@ -7,12 +7,7 @@
 import greencity.constant.AppConstant;
 import greencity.constant.ErrorMessage;
 import greencity.constant.OrderHistory;
-import greencity.dto.AllActiveLocationsDto;
-import greencity.dto.CreateAddressRequestDto;
-import greencity.dto.LocationsDtos;
-import greencity.dto.OrderCourierPopUpDto;
-import greencity.dto.RegionDto;
-import greencity.dto.TariffsForLocationDto;
+import greencity.dto.*;
```
Comment (DenysLiubchenko) [inline service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java]: Wildcart import

File: service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java [hunk 6]
```diff
@@ -148,50 +144,7 @@
...
-import static greencity.constant.ErrorMessage.TARIFF_NOT_FOUND;
-import static greencity.constant.ErrorMessage.TARIFF_OR_LOCATION_IS_DEACTIVATED;
-import static greencity.constant.ErrorMessage.THE_SET_OF_UBS_USER_DATA_DOES_NOT_EXIST;
-import static greencity.constant.ErrorMessage.TOO_MANY_CERTIFICATES;
-import static greencity.constant.ErrorMessage.TOO_MUCH_POINTS_FOR_ORDER;
-import static greencity.constant.ErrorMessage.TO_MUCH_BAG_EXCEPTION;
-import static greencity.constant.ErrorMessage.USER_DONT_HAVE_ENOUGH_POINTS;
-import static greencity.constant.ErrorMessage.USER_WITH_CURRENT_ID_DOES_NOT_EXIST;
-import static greencity.constant.ErrorMessage.USER_WITH_CURRENT_UUID_DOES_NOT_EXIST;
+import static greencity.constant.ErrorMessage.*;
```
Comment (DenysLiubchenko) [inline service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java]: Wildcart import

File: service/src/main/java/greencity/service/ubs/UBSManagementEmployeeServiceImpl.java [hunk 7]
```diff
@@ -21,14 +23,10 @@
...
+import greencity.exceptions.user.UserNotFoundException;
 import greencity.filters.EmployeeFilterCriteria;
 import greencity.filters.EmployeePage;
-import greencity.repository.EmployeeCriteriaRepository;
-import greencity.repository.EmployeeRepository;
-import greencity.repository.PositionRepository;
-import greencity.repository.ReceivingStationRepository;
-import greencity.repository.TariffsInfoRepository;
-import greencity.repository.UserRepository;
+import greencity.repository.*;
```
Comment (DenysLiubchenko) [inline service/src/main/java/greencity/service/ubs/UBSManagementEmployeeServiceImpl.java]: Wildcart import

File: service/src/test/java/greencity/ModelUtils.java [hunk 8]
```diff
@@ -102,11 +102,7 @@
 import greencity.dto.service.ServiceDto;
 import greencity.dto.service.TariffServiceDto;
 import greencity.dto.table.ColumnWidthDto;
-import greencity.dto.tariff.EditTariffDto;
-import greencity.dto.tariff.GetTariffInfoForEmployeeDto;
-import greencity.dto.tariff.GetTariffLimitsDto;
-import greencity.dto.tariff.GetTariffsInfoDto;
-import greencity.dto.tariff.SetTariffLimitsDto;
+import greencity.dto.tariff.*;
```
Comment (DenysLiubchenko) [inline service/src/test/java/greencity/ModelUtils.java:105]: Wildcart import

**Review by DenysLiubchenko — CHANGES_REQUESTED** [review]

**Review by HelenSotnik — APPROVED** [review]

**Review by DenysLiubchenko — APPROVED** [review]

**Pull request merged**